crates/chainspec/src/spec.rs

```diff
@@ -746,7 +720,7 @@ impl From<Genesis> for ChainSpec {
         ordered_hardforks.append(&mut hardforks);
 
         // Extract blob parameters directly from blob_schedule
-        let blob_params = HardforkBlobParams::from_schedule(&genesis.config.blob_schedule);
+        let blob_params = BlobScheduleBlobParams::from_schedule(&genesis.config.blob_schedule);
```

Comment:
this can now be replaced with 

https://github.com/alloy-rs/alloy/blob/16b48bfe3cce6fc47313d425ce80432128d9bd64/crates/genesis/src/lib.rs#L501-L501